docs/Configuration.rst

```diff
@@ -5317,9 +5317,9 @@ then the query is sent as if it is two words.
 
 .. _Configuration.ElasticSearch:
   
-=========================================
-Elasticsearch Embedded mode Configuration
-=========================================
+=============================
+Elasticsearch Configuration
+=============================
```

Comment:
The text below should also be moved into the chapter "Properties of the Elasticsearch embedded node" IMO since it is only related to embedded mode.

Reply:
OK.

Reply:
done